airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -606,7 +606,7 @@
 - name: Jira
   sourceDefinitionId: 68e63de2-bb83-4c7e-93fa-a8a9051e3993
   dockerRepository: airbyte/source-jira
-  dockerImageTag: 0.2.22
+  dockerImageTag: 0.2.23
```

Comment:
@Barsoomx do not edit this file, we will do this change when we publish. Please revert.